tests/test_mps.py

```diff
@@ -18,15 +20,42 @@
 )
 
 
+def test_libhandle_manager() -> None:
+    circ = Circuit(5)
+    mps = MPS(qubits=circ.qubits)
+
+    # Catch exception due to no library handle initialised
+    exception_caught = False
+    try:
+        mps.vdot(mps)
+    except RuntimeError:
+        exception_caught = True
+    assert exception_caught

```

Comment:
You can do it in pytest nicer as follows:
```
with pytest.raises(RuntimeError):
    mps.vdot(mps)
```

Reply:
That's very handy, thanks!